tests/Agent/IntegrationTests/UnboundedIntegrationTests/UnboundedIntegrationTests.csproj

```diff
@@ -76,4 +76,4 @@
     <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
   </ItemGroup>
 
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
I can't add a comment, but can we remove:
`<PackageReference Include="MySqlConnector" Version="1.0.1" />`
from line 41?